vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -2064,6 +2065,20 @@ else if (GameModule.MODULE_CURRENT_LOCALE.equals(key)) {
     else if (GameModule.MODULE_CURRENT_LOCALE_NAME.equals(key)) {
       return Resources.getLocale().getDisplayName();
     }
+
+    //BR// MapName_isVisible property for each map window
+    for (final Map map : Map.getMapList()) {
+      if ((map.getConfigureName() + "_isVisible").equals(key) || (map.getConfigureName().replaceAll(" ", "_") + "_isVisible").equals(key)) { //NON-NLS
+        final Container tla = (map.getComponent() != null) ? ((JPanel)map.getComponent()).getTopLevelAncestor() : null;
+        if ((tla != null) && tla.isVisible()) {
+          return "true"; //NON-NLS
+        }
+        else {
+          return "false"; //NON-NLS
+        }
```

Comment:
`return String.valueOf(tla != null && tla.isVisible());` could replace this block.

Reply:
Also, do you want `isShowing()` instead of `isVisible()`? I ran into this when fixing some of those `IllegalStateException`s with components that aren't displayed.

Reply:
I've tested this out, works well. I believe for Map views, isVisible() will be equivalent to isShowing(), but isShowing() is definitely safer.